### PR TITLE
feat:vehicles miles report

### PR DESCRIPTION
**Pull request by saurkarankita**
## Change Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by FH-sandeepkaur1** [comment]
Please resolve conflicts @saurkarankita

File: src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.css [hunk 1]
```diff
@@ -1,9 +0,0 @@
-ngx-datatable {
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.css:1]: Why you have delete this? @AmitojSingh1 Please review

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
@AmitojSingh1 Please review

File: src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.css (hunk 1 above)
Comment (saurkarankita) [reply src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.css:1]: @kunalfleethawks  @AmitojSingh1  when using this css scroll bar code. infinite pagination had not worked properly, that's why we need to remove this, then infinite pagination is working perfectly,

**Pull request closed without merging**